clients/python-wrapper/tests/utests/test_tag.py

```diff
@@ -0,0 +1,13 @@
+from lakefs.repository import Repository
+from tests.utests.common import get_test_client
+
+
+def test_tag_creation():
```

Comment:
test_tag_object? we don't create a tag here

Reply:
This is a stub until we implement the Tag class